python/lib/modeldata/dmod/test/test_bmi_init_config_auto_generator.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+import pandas as pd
+import geopandas as gpd
+
+from pathlib import Path
+from ..modeldata.data.bmi_init_config_auto_generator import BmiInitConfigAutoGenerator, NgenRealization
+from typing import Dict
+from ngen.init_config.serializer_deserializer import (IniSerializerDeserializer, JsonSerializerDeserializer,
+                                                      NamelistSerializerDeserializer, YamlSerializerDeserializer)
+
+
+@unittest.skip
+class TestBmiInitConfigAutoGenerator(unittest.TestCase):
+
+    @staticmethod
+    def _remove_dir(dir_path: Path):
+        for p in dir_path.iterdir():
+            if p.is_dir():
+                TestBmiInitConfigAutoGenerator._remove_dir(p)
+            else:
+                p.unlink()
+        dir_path.rmdir()
+
+    @staticmethod
+    def _prep_output_dir(output_dir: Path):
+        if output_dir.is_dir():
+            TestBmiInitConfigAutoGenerator._remove_dir(output_dir)
+            output_dir.mkdir()
+        elif not output_dir.exists():
+            output_dir.mkdir()
+        else:
+            raise RuntimeError(f"Test output directory path {output_dir} is existing non-directory")
+
+    def setUp(self):
+        self.generators: Dict[int, BmiInitConfigAutoGenerator] = dict()
+        self.output_dirs: Dict[int, Path] = dict()
+
+        # TODO: (later) find a better way to do this
+        hf_gpkg = Path.home().joinpath("Developer/noaa/data/hydrofabric/v201/nextgen_01.gpkg")
+        hf_model_data = Path.home().joinpath("Developer/noaa/data/hydrofabric/v201/nextgen_01.parquet")
+        real_cfg_file = Path.home().joinpath(
+            "Developer/noaa/dmod/data/example_realization_configs/ex_realization_config_03.json")
+        noah_params_dir = Path.home().joinpath("Developer/noaa/data/noah_owp_ex_params_dir_1")
+
+        self.output_dirs[0] = hf_model_data.parent.joinpath(f"{self.__class__.__name__}_out_0")
```

Comment:
Hmm, well, as discussed above, I'd planned on revisiting the tests a bit later, and these paths might be revised then also.  For the moment at least, since the tests are more manual and incomplete, this was to make it easier to go examine the created files.